README.md

```diff
@@ -7,17 +7,40 @@
 - [Contributing](#contributing)
 - [Metrics](#metrics)
 - [Contributors](#contributors)
+- [Acknoledgements](#Acknoledgements)
 - [License](#license)
 
 ## Introduction
 
+### Scope
+
+A well-designed Open Source Program Office (OSPO), when present, is the center of competency for an organization’s open source operations and structure. As described in the [OSPO deep dive research report](https://www.linuxfoundation.org/research/a-deep-dive-into-open-source-program-offices), OSPOs benefit organizations worldwide by:
+
+* Implementing unique and flexible sets of tools that support OSS development models while meeting corporate Information Technology guidelines
+* Overseeing the establishment or adaptation of internal policies to better manage open source software (OSS) compliance in fast-moving, dynamic environments
+* Helping to bridge the cultural gap between traditional software development practices and the requirements of open source development
+* Improving technical mentorship, and compliance-related education and training programming for team members across all levels of the organization
+
+> These strategic open source centers can take many shapes. For instance, depending on the size of the organization, its structure can vary from a virtual OSPO with one FTE to a large dedicated team. To learn more, please take a look at the different [structures](https://8112310.fs1.hubspotusercontent-na1.net/hubfs/8112310/LF%20Research/LFR_LFAID_Deep_Dive_Open_Source_Program_Offices_081922.pdf), [roles](https://8112310.fs1.hubspotusercontent-na1.net/hubfs/8112310/LF%20Research/LFR_LFAID_Deep_Dive_Open_Source_Program_Offices_081922.pdf), and [responsibilities](https://8112310.fs1.hubspotusercontent-na1.net/hubfs/8112310/LF%20Research/LFR_LFAID_Deep_Dive_Open_Source_Program_Offices_081922.pdf) an organization can choose when builging an OSPO.
+
+### Problem
+
+Tracking performance metrics is one of the [key responsibilities of many OSPOs](https://ospomindmap.todogroup.org/): [OSPOs might use specialized tools](https://todogroup.org/guides/management-tools/) to track their organization’s contributions to open source projects, analyze the type of contributions from their organization, identify contribution patterns, and provide recommendations to improve the development impact
+
 ### Goals
 
-Community initiatives do not reflect the same value as other common organizational efforts. This results in community work often being undervalued by people, organizations, communities, and society. The Value working group aims to advance how we understand the value that projects can provide. 
+Community initiatives do not reflect the same value as organizational efforts. The OSPO Metrics working group aims to advance how organizations understand the value that open source projects can provide as well as the value of these programs / initiatives.
+
+Main objectives include:
+
+* Define useful metrics
```

Comment:
Maybe add more clarity on the scope as to metrics from whom something like "Define useful metrics for different stakeholders in OSPOs" or simply  " Define useful metrics for OSPOs"